src/frontend/components/FullPage.jsx

```diff
@@ -29,10 +19,33 @@ class FullPage extends Component {
     this.props.loadSession();
   }
 
+  renderNavbar() {
+    const loggedLinks = [
+      { label: 'Home', class: 'navigation', type: 'link', to: '/home' },
+      { label: 'Postar Pergunta', class: 'navigation', type: 'link', to: '/postar-pergunta' },
+      { label: 'Logout', class: 'action', type: 'button', action: () => this.props.logout() },
+      {
+        label: 'Perfil',
+        class: 'action',
+        type: 'button',
+        action: () => navigate('/usuarios/' + this.props.session.usuarioId),
+      },
+    ];
+
+    const unloggedLinks = [
+      { label: 'Login', class: 'navigation', type: 'link', to: '/login' },
+      { label: 'Registrar-se', class: 'navigation', type: 'link', to: '/registro-usuario' },
+    ];
+    if (this.props.session) {
+      return <Navbar key={'nav-logged'} to="/home" title={'Não Faço a Menor Ideia!'} links={loggedLinks} />;
```

Comment:
`return <Navbar key={'nav-logged'} to="/home" title={'Não Faço a Menor Ideia!'} links={this.props.session ? loggedLinks : unloggedLinks} />;`